experimental/packages/opentelemetry-sdk-metrics-base/src/Measurement.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api'
+import { Attributes } from '@opentelemetry/api-metrics'
+
+export type Measurement = {
+    value: number;
+    // TODO use common attributes
+    attributes: Attributes
+    context?: api.Context;
```

Comment:
we could add a todo for examplar here ?

Reply:
The API spec of measurement doesn't specify exemplar: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#measurement

And there is no SDK measurement defined in spec. We will have to implement exemplars in the future and it may require adding them to this interface or it may not so I would prefer to leave it out until we know it is needed.